core/os/android/adb/commands.go

```diff
@@ -443,7 +442,9 @@ func (b *binding) resolveDriverPath(ctx context.Context, driver string) (Driver,
 		return Driver{}, nil
 	}
 	path = path[8:]
-	if path == "" {
+	// If the driver package path doesn't have the /data/app prefix, it means the returned
+	// path is the preinstalled emtpy prerelease driver.
+	if path == "" || !strings.HasPrefix(path, "/data/app") {
```

Comment:
Should we do this the other way around, i.e. `if strings.HasPrefix(path, "/vendor/app")` ignore it?

Reply:
It is vendor dependent, vendors can put it inside `/system/priv-app` as well.

Reply:
Marking this as resolved. :)